From b40c9856738f33c235fc0c774b405414e4078feb Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:10:03 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit 398236e4f03e526d8e56273f9fc2148d80b10969
Merge: 7e0410fc 2276e988
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Jul 20 23:50:44 2018 +0000

    Snap for 4905103 from 2276e988af6d7b4d14b0b8d479ff6d8436522d1c to pi-dr1-release

    Change-Id: I665f861777d4d94e4f7abea7b40355c1b2386035

commit 2276e988af6d7b4d14b0b8d479ff6d8436522d1c
Author: Bill Yi <byi@google.com>
Date:   Fri Jul 20 11:18:47 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I62dc8129e9dfc3d787c4ff557dffadfdd8986d81

commit 7e0410fcf4950aca0a92a29930bf7b8295a84bad
Merge: f023911e 9bdea89d
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:01:25 2018 +0000

    Snap for 4859140 from 9bdea89d27d9281a50e1ad6bed73195db899e0cc to pi-dr1-release

    Change-Id: Id381898806f48d182d87bbdb48cd46a516c927d7

commit 9bdea89d27d9281a50e1ad6bed73195db899e0cc
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 07:23:31 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I3b652ffe1f16fd2a3305f93449c7191441a2e0dc

Change-Id: I8ab8ca35e875f06a73a21fbd5dfcc7ce507742c7
---
 res/values-bn/strings.xml | 4 ++--
 res/values-nl/strings.xml | 4 ++--
 res/values-th/strings.xml | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/res/values-bn/strings.xml b/res/values-bn/strings.xml
index 3ab11f74..891a74f0 100644
--- a/res/values-bn/strings.xml
+++ b/res/values-bn/strings.xml
@@ -28,8 +28,8 @@
     <string name="the_following_is_your_mdm_for_device" msgid="6717973404364414816">"নিম্নলিখিত অ্যাপ্লিকেশানটি আপনার ডিভাইস পরিচালনা করবে:"</string>
     <string name="next" msgid="1004321437324424398">"পরবর্তী"</string>
     <string name="setting_up_workspace" msgid="7862472373642601041">"কর্মস্থলের প্রোফাইল সেট-আপ করা হচ্ছে…"</string>
-    <string name="admin_has_ability_to_monitor_profile" msgid="4552308842716093826">"আপনার প্রশাসক আপনার প্রোফাইলের সেটিংস, কর্পোরেট অ্যাক্সেস, অ্যাপ, অনুমতি, এবং নেটওয়ার্ক কার্যকলাপ সহ আপনার প্রোফাইলের সাথে সম্পর্কিত ডেটা, এবং আপনার ডিভাইসের লোকেশন, এবং কল ও পরিচিতি অনুসন্ধানের ইতিহাসের উপরে নজর রাখতে ও সেগুলি পরিচালনা করতে পারেন।<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>আপনার প্রতিষ্ঠানের গোপনীয়তা নীতি সহ আরও তথ্যের জন্য আপনার প্রশাসকের সাথে যোগাযোগ করুন।"</string>
-    <string name="admin_has_ability_to_monitor_device" msgid="8598573866079474861">"আপনার প্রশাসক আপনার ডিভাইসের সেটিংস, কর্পোরেট অ্যাক্সেস, অ্যাপ, অনুমতি, এবং নেটওয়ার্ক কার্যকলাপ সহ ডিভাইসের সাথে সম্পর্কিত ডেটা, ডিভাইসের লোকেশন, এবং কল ও পরিচিতি অনুসন্ধানের ইতিহাসের উপরে নজর রাখতে ও সেগুলি পরিচালনা করতে পারেন।<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>আপনার প্রতিষ্ঠানের গোপনীয়তা নীতি সহ আরও তথ্যের জন্য আপনার প্রশাসকের সাথে যোগাযোগ করুন।"</string>
+    <string name="admin_has_ability_to_monitor_profile" msgid="4552308842716093826">"আপনার প্রশাসক আপনার প্রোফাইলের সেটিংস, কর্পোরেট অ্যাক্সেস, অ্যাপ, অনুমতি, এবং নেটওয়ার্ক অ্যাক্টিভিটি সহ আপনার প্রোফাইলের সাথে সম্পর্কিত ডেটা, এবং আপনার ডিভাইসের লোকেশন, এবং কল ও পরিচিতি সার্চের ইতিহাসের উপরে নজর রাখতে ও সেগুলি পরিচালনা করতে পারেন।<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>আপনার প্রতিষ্ঠানের গোপনীয়তা নীতি সহ আরও তথ্যের জন্য আপনার প্রশাসকের সাথে যোগাযোগ করুন।"</string>
+    <string name="admin_has_ability_to_monitor_device" msgid="8598573866079474861">"আপনার প্রশাসক আপনার ডিভাইসের সেটিংস, কর্পোরেট অ্যাক্সেস, অ্যাপ, অনুমতি, এবং নেটওয়ার্ক অ্যাক্টিভিটি সহ ডিভাইসের সাথে সম্পর্কিত ডেটা, ডিভাইসের লোকেশন, এবং কল ও পরিচিতি সার্চের ইতিহাসের উপরে নজর রাখতে ও সেগুলি পরিচালনা করতে পারেন।<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>আপনার প্রতিষ্ঠানের গোপনীয়তা নীতি সহ আরও তথ্যের জন্য আপনার প্রশাসকের সাথে যোগাযোগ করুন।"</string>
     <string name="theft_protection_disabled_warning" msgid="3708092473574738478">"চুরির-সুরক্ষা বৈশিষ্ট্যগুলি ব্যবহার করতে আপনার ডিভাইসে অবশ্যই একটি পাসওয়ার্ড দ্বারা সুরক্ষিত স্ক্রিন লক ব্যবস্থাটিকে সক্ষম রাখতে হবে৷"</string>
     <string name="contact_your_admin_for_more_info" msgid="5959191345827902911">"আপনার প্রতিষ্ঠানের গোপনীয়তা নীতিগুলি সহ আরও তথ্যের জন্য আপনার প্রশাসকের সাথে যোগাযোগ করুন।"</string>
     <string name="learn_more_link" msgid="3012495805919550043">"আরও জানুন"</string>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index 0044aacf..706cfd09 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -28,8 +28,8 @@
     <string name="the_following_is_your_mdm_for_device" msgid="6717973404364414816">"Je apparaat wordt beheerd met de volgende app:"</string>
     <string name="next" msgid="1004321437324424398">"Volgende"</string>
     <string name="setting_up_workspace" msgid="7862472373642601041">"Werkprofiel instellen…"</string>
-    <string name="admin_has_ability_to_monitor_profile" msgid="4552308842716093826">"Je beheerder kan instellingen, zakelijke toegang, apps, machtigingen en gegevens die zijn gekoppeld aan dit profiel (waaronder je netwerkactiviteit) bijhouden en beheren, evenals de locatie, oproepgeschiedenis en zoekgeschiedenis voor contacten van je apparaat.<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>Neem contact op met je beheerder voor meer informatie en bij vragen over het privacybeleid van je organisatie."</string>
-    <string name="admin_has_ability_to_monitor_device" msgid="8598573866079474861">"Je beheerder kan instellingen, zakelijke toegang, apps, machtigingen en gegevens die zijn gekoppeld aan dit apparaat (waaronder je netwerkactiviteit) bijhouden en beheren, evenals de locatie, oproepgeschiedenis en zoekgeschiedenis voor contacten van je apparaat.<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>Neem contact op met je beheerder voor meer informatie en bij vragen over het privacybeleid van je organisatie."</string>
+    <string name="admin_has_ability_to_monitor_profile" msgid="4552308842716093826">"Je beheerder kan instellingen, zakelijke toegang, apps, machtigingen en gegevens die zijn gekoppeld aan dit profiel (waaronder je netwerkactiviteit) bijhouden en beheren, evenals de locatie, gespreksgeschiedenis en zoekgeschiedenis voor contacten van je apparaat.<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>Neem contact op met je beheerder voor meer informatie en bij vragen over het privacybeleid van je organisatie."</string>
+    <string name="admin_has_ability_to_monitor_device" msgid="8598573866079474861">"Je beheerder kan instellingen, zakelijke toegang, apps, machtigingen en gegevens die zijn gekoppeld aan dit apparaat (waaronder je netwerkactiviteit) bijhouden en beheren, evenals de locatie, gespreksgeschiedenis en zoekgeschiedenis voor contacten van je apparaat.<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>Neem contact op met je beheerder voor meer informatie en bij vragen over het privacybeleid van je organisatie."</string>
     <string name="theft_protection_disabled_warning" msgid="3708092473574738478">"Als je diefstalbeveiligingsfuncties wilt gebruiken, moet je een met een wachtwoord beveiligde schermvergrendeling hebben ingesteld voor je apparaat."</string>
     <string name="contact_your_admin_for_more_info" msgid="5959191345827902911">"Neem contact op met je beheerder voor meer informatie, waaronder het privacybeleid van je organisatie."</string>
     <string name="learn_more_link" msgid="3012495805919550043">"Meer informatie"</string>
diff --git a/res/values-th/strings.xml b/res/values-th/strings.xml
index b07baa62..5a2f5c07 100644
--- a/res/values-th/strings.xml
+++ b/res/values-th/strings.xml
@@ -32,7 +32,7 @@
     <string name="admin_has_ability_to_monitor_device" msgid="8598573866079474861">"ผู้ดูแลระบบสามารถตรวจสอบและจัดการการตั้งค่า การเข้าถึงของบริษัท แอป สิทธิ์ และข้อมูลที่เชื่อมโยงกับอุปกรณ์นี้ ซึ่งรวมถึงกิจกรรมในเครือข่ายของคุณ ตลอดจนตำแหน่งของอุปกรณ์ ประวัติการโทร และประวัติการค้นหารายชื่อติดต่อ<xliff:g id="LINE_BREAK">&lt;br&gt;&lt;br&gt;</xliff:g>โปรดติดต่อผู้ดูแลระบบเพื่อสอบถามข้อมูลเพิ่มเติม รวมทั้งนโยบายความเป็นส่วนตัวขององค์กร"</string>
     <string name="theft_protection_disabled_warning" msgid="3708092473574738478">"หากต้องการใช้ฟีเจอร์การป้องกันขโมย อุปกรณ์ของคุณต้องมีการล็อกหน้าจอที่ป้องกันด้วยรหัสผ่าน"</string>
     <string name="contact_your_admin_for_more_info" msgid="5959191345827902911">"ติดต่อผู้ดูแลระบบสำหรับข้อมูลเพิ่มเติม รวมถึงนโยบายความเป็นส่วนตัวขององค์กร"</string>
-    <string name="learn_more_link" msgid="3012495805919550043">"เรียนรู้เพิ่มเติม"</string>
+    <string name="learn_more_link" msgid="3012495805919550043">"ดูข้อมูลเพิ่มเติม"</string>
     <string name="cancel_setup" msgid="2949928239276274745">"ยกเลิก"</string>
     <string name="ok_setup" msgid="4593707675416137504">"ตกลง"</string>
     <string name="user_consent_msg" msgid="8820951802130353584">"ฉันยินยอม"</string>
@@ -136,7 +136,7 @@
     <string name="setup_device" msgid="6725265673245816366">"ตั้งค่าอุปกรณ์"</string>
     <string name="setup_device_encryption" msgid="7852944465414197103">"ตั้งค่าอุปกรณ์ การเข้ารหัส"</string>
     <string name="setup_device_progress" msgid="7035335208571175393">"ตั้งค่าอุปกรณ์ กำลังแสดงความคืบหน้า"</string>
-    <string name="learn_more_label" msgid="2723716758654655009">"ปุ่มเรียนรู้เพิ่มเติม"</string>
+    <string name="learn_more_label" msgid="2723716758654655009">"ปุ่มดูข้อมูลเพิ่มเติม"</string>
     <string name="mdm_icon_label" msgid="3399134595549660561">"ไอคอน <xliff:g id="ICON_LABEL">%1$s</xliff:g>"</string>
     <string name="section_heading" msgid="3924666803774291908">"หัวข้อของส่วน <xliff:g id="SECTION_HEADING">%1$s</xliff:g>"</string>
     <string name="section_content" msgid="8875502515704374394">"เนื้อหาของส่วน <xliff:g id="SECTION_HEADING">%1$s</xliff:g> คือ <xliff:g id="SECTION_CONTENT">%2$s</xliff:g>"</string>
-- 
2.17.1

